src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        if (email.equals("bob@i.ua") & (password.equals("1234")) || (email.equals("alice@i.ua"))){
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Not sure if it follows task requirements. Please, consider implementing `UserService.findByEmail `logic. After that try to guess how we can use our implemented `UserService` in the authentication context.